bazel/BUILD

```diff
@@ -166,6 +171,8 @@ config_setting(
 # Alias pointing to the selected version of BoringSSL:
 # - BoringSSL FIPS from @boringssl_fips//:ssl,
 # - non-FIPS BoringSSL from @boringssl//:ssl.
+# In case of disabled BoringSSL the @boringssl repository can be remapped to an
+# alternative one in WORKSPACE of the corresponding third party TLS extension.
```

Comment:
I still think that the remapping suggestion is wrong, so I'd prefer if you removed it.